docs/learn/export-pipeline/data-value-extractor.md

```diff
@@ -33,39 +33,79 @@ import WIP from '../\_wip.md';
 
 ## Basic usage
 
-The Data Value Extractor serves to extract data from a document that is linked in annotation's metadata. The main use case is to process data from (reference rest-api-export) as a part of (reference export-pipeline).
+The Data Value Extractor serves to extract data from a document that is linked in annotation's metadata. The main use case is to process data from [REST API Export](https://rossum.university/docs/learn/export-pipeline/rest-api-export) as a part of the [Export Pipeline](https://rossum.university/docs/learn/export-pipeline).
```

Comment:
I've refactored the links here: https://github.com/rossumai/university/pull/480/files#diff-a001c338961139c7557bd61e5a8a7b7bbc9de79bfb7c1a20df037f670889895fR11

It is better to use the relative links instead of the absolute URLs because then we can detect whether all links work correctly (our CI tests will stop any broken links).